admin_guide/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -0,0 +1,182 @@
+= Persistent Storage Using GlusterFS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+OpenShift can utilize persistent storage using Distributed File Systems (DFS) like GlusterFS.  Some familiarity with Kuberenetes/Docker is assumed.  Also assumed that there is access to an existing GlusterFS cluster and volume and that the glusterfs-client has been installed on all OpenShift nodes in the cluster.
+
+The Kubernetes link:../dev_guide/persistent_volumes.html[persistent volume]
+framework allows administrators to provision a cluster with persistent storage
+and gives users a way to request those resources without having any knowledge of
+the underlying infrastructure.
+
+[NOTE]
+====
+`Persistent Volumes` are not bound to a single project/namespace, they can be shared across the OpenShift environment, where as, `Persistent Volume Claims` are project/namespace specific and can be requested by users
+====
+
+
+[IMPORTANT]
+====
+High-availability of storage in the infrastructure is left to the underlying
+storage provider.
+====
+
+
+[[provisioning]]
+
+== Provisioning
+Storage must exist in the underlying infrastructure before it can be mounted as
+a volume in OpenShift. All that is required for GlusterFS is a distinct list of
+servers in the Gluster cluster (endpoints), Gluster Volumes (PersistentVolume) and the `*PersistentVolume*` API.
+
+
+[[create-gluster-endpoints]]
+
+=== Create Gluster Endpoints
+
+.Persistent Volume Endpoints Definition
+====
+
+For this example, you will have to define the GlusterFS Cluster as “EndPoints” within Kubernets/OpenShift platform using a file similar to [endpoints configuration file](gluster-endpoints.json).  You must define the IP/Hostname of your gluster servers and the port that you want to use.  The port value can be any numeric value within the accepted range of ports.
```

Comment:
Were you planning on including a link to an actual `gluster-endpoints.json` file, or is this basically just referring to the example JSON that now follows?